src/app/components/mnoe-api/marketplace.svc.coffee

```diff
@@ -20,7 +20,7 @@ angular.module 'mnoEnterpriseAngular'
 
     @addAppReview = (appId, data) ->
       payload = {app_review: data}
-      MnoeApiSvc.one('marketplace', parseInt(appId)).post('app_review', payload).then(
+      MnoeApiSvc.one('marketplace', parseInt(appId)).doPOST(payload, '/app_review').then(
```

Comment:
@alexnoox ,thanks a lot. You can check that there is a new commit with that change added.